package.json

```diff
@@ -12,7 +12,8 @@
   ],
   "scripts": {
     "tsc": "tsc -p tsconfig.json",
-    "dev": "tsx src/bin.ts",
+    "remove-built-files": "node removeJsFiles.js",
```

Comment:
@moskalyk Can you remove this one, and the `removeJsFiles` – I am not comfortable with it and I don't think it's needed.

Reply:
Yeah sure